kong-0.2.0-1.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong"
-version = "0.1.1beta-2"
+version = "0.2.0-1"
 supported_platforms = {"linux", "macosx"}
 source = {
   url = "git://github.com/Mashape/kong",
-  tag = "0.1.1beta-2"
```

Comment:
Oh so that's a good news!


> On Apr 3, 2015, at 6:44 AM, Marco Palladino <notifications@github.com> wrote:
> 
> In kong-0.2.0-1.rockspec:
> 
> >  supported_platforms = {"linux", "macosx"}
> >  source = {
> >    url = "git://github.com/Mashape/kong",
> > -  tag = "0.1.1beta-2"
> This is from the Luarocks doc
> 
> version (string, mandatory field) - the version of the package, plus a suffix indicating the revision of the rockspec. Example: "2.0.1-1
> 
> They consider the -1 the revision.
> 
> —
> Reply to this email directly or view it on GitHub.
>